app/jobs/enqueue_publishers_for_payout_job.rb

```diff
@@ -15,14 +17,22 @@ def perform(should_send_notifications: false, final: true, payout_report_id: "",
       payout_report = PayoutReport.find(payout_report_id)
     else
       payout_report = PayoutReport.create(final: final,
+                                          manual: manual,
                                           fee_rate: fee_rate,
                                           expected_num_payments: PayoutReport.expected_num_payments(publishers))
     end
 
     publishers.find_each do |publisher|
-      IncludePublisherInPayoutReportJob.perform_later(payout_report_id: payout_report.id,
-                                                      publisher_id: publisher.id,
-                                                      should_send_notifications: should_send_notifications)
+      if manual 
+        # We can consider using a job here if n is sufficiently large 
+        ManualPayoutReportPublisherIncluder.new(publisher: publisher,
```

Comment:
Also make this a perform_later otherwise we'll have a blocking call on the view

Reply:
I have considered that as well, we should consider using a job here when `n` becomes sufficiently large.